assets/js/components/settings/SettingsAdmin.js

```diff
@@ -43,6 +44,9 @@ import { useFeature } from '../../hooks/useFeature';
 export default function SettingsAdmin() {
 	const audienceSegmentationEnabled = useFeature( 'audienceSegmentation' );
 
+	const configuredAudiences = useSelect( ( select ) =>
+		select( CORE_USER ).getConfiguredAudiences()
```

Comment:
Hi @ankitrox, this change has introduced a regression as reported by @wpdarren in [Slack](https://fueled10up.slack.com/archives/CBKKQEBR9/p1726753396158329).

We need to guard against the request to the `audience-settings` endpoint with a check for `audienceSegmentationEnabled` here:

```suggestion
	const configuredAudiences = useSelect(
		( select ) =>
			audienceSegmentationEnabled &&
			select( CORE_USER ).getConfiguredAudiences()
```

Please can you raise a followup PR to fix it, targeting the `main` branch?